packages/mui-material/src/TextField/TextField.test.js

```diff
@@ -121,6 +121,14 @@ describe('<TextField />', () => {
         outlinedInputClasses.notchedOutline,
       );
     });
+    it('should render `0` label properly', () => {
```

Comment:
Should we add the test from the `NotchedOutline` here too?

Reply:
It would not be necessary for now as it's been checked in `NotchedOutline` component but for the possible future changes or new components that might be introduced, I think it is a good idea to add the test to the `TextField` as well. I will do it